glib/src/subclass/interface.rs

```diff
@@ -47,35 +47,12 @@ macro_rules! prerequisite_list_trait_impl(
     ($($name:ident),+) => (
         impl<$($name: crate::ObjectType),+> PrerequisiteList for ( $($name),+ ) {
             fn types() -> Vec<ffi::GType> {
-                let mut types = Vec::new();
-                prerequisite_list_trait_inner!(types, $($name)+)
+                vec![$($name::static_type().to_glib()),+]
```

Comment:
Oh... yes that's easier :)

Reply:
You can do the same simplification in src/subclass/types.rs for the `InterfaceList`

Reply:
@GuillaumeGomez Are you sending a PR for this one?

Reply:
Oh sorry, didn't see your message. Doing it right away.